apps/challenge-registry/challenge-service/src/main/java/org/sagebionetworks/challenge/model/dto/ChallengeDto.java

```diff
@@ -101,6 +107,55 @@ public void setName(String name) {
     this.name = name;
   }
 
+  public ChallengeDto headline(String headline) {
+    this.headline = headline;
+    return this;
+  }
+
+  /**
+   * The headline of the a challenge
+   *
+   * @return headline
+   */
+  @Size(min = 0, max = 80)
+  @Schema(
+      name = "headline",
+      example = "Example challenge headline",
+      description = "The headline of the a challenge",
+      required = false)
+  public String getHeadline() {
```

Comment:
Ah, back to the good ol' getters and setters -- it's been awhile!

Reply:
They are even better when they are automatically generated like here!